contracts/examples/ping-pong-egld/snippets.sh

```diff
@@ -0,0 +1,56 @@
+ALICE="${USERS}/alice.pem"
```

Comment:
did you get to play around with the snippets?
if yes -> great! we keep it
if no, just copied it from somewhere else -> better leave this out, this contract will be used by other devs as template, better not confuse them, we'll have time for snippets later

Reply:
I didn't really use it. I removed it now